src/main/java/ro/vladutit/Don/t/forget/v2/controller/CategoryController.java

```diff
@@ -46,10 +46,10 @@ public String updateItemForm(@PathVariable(value = "id") Long id, Model model) {
     }
 
     // Delete a category by id
-    @RequestMapping("/categoryd/{id}")
-    public String deleteItem(@PathVariable (value = "id") Long id) {
+    @RequestMapping(value = "/categoryd/{id}", method = RequestMethod.DELETE)
```

Comment:
categoryd ??

Reply:
It's a discussion I've had before. If I have a longer link ex: category / delete / id something happens with the css part and images no longer work. On the ITEM side, I managed to put in the controller as the main and common route on / item /. Here I can't do the same thing, if I put / category /, then the same links remain to the item and category: delete / id and then he doesn't know which to turn to, if it's category or item. I will analyze the subject, but at the moment I have chosen this option, so as not to stay too long in place with the project, but the subject is noted in my TODO list.